files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -306,6 +306,12 @@
     path: /etc/ssh/sshd_config
     regexp: '^(Subsystem.*sftp.*)'
     replace: '#\1'
+  when:
+    #
+    # For developer convenience, we only enable the sftp
```

Comment:
The comment could be a little bit confusing since this ansible block doesn't enable sftp, but disables it. I'd suggest a slightly different wording:
We disable sftp on external variants, and leave it enabled on internal variants for developer convenience and to facilitate test automation.

Reply:
Yeah, I guess that's better, Thanks Seb!! Updated